tutorials/3_RunningBundleOfRuns.md

```diff
@@ -3,36 +3,56 @@ Running more than one REMIND scenario
 * `path_gdx_ref` points to the run used for all `t < cm_startyear`, which can be used for example for delayed transition scenarios, and which is also used as a comparison for the policy cost calculation.
 * `path_gdx_bau` points to the run used as business as usual (BAU) scenario, for example for runs using [`45_carbonprice/NDC`](../modules/45_carbonprice/NDC), where some countries specify emission as percentage reduction compared to BAU.
 
-These three columns starting with `path…` can either be left empty or used to point to either finished runs or other rows of the current model execution:
+These three columns starting with `path…` can point to either finished runs or other rows of the current model execution:
 * provide a path to an existing `gdx` file such as `./output/SSP2-Base_2021-12-24_19.30.00/fulldata.gdx`. For subfolders of `./output/`, writing the folder name `SSP2-Base_2021-12-24_19.30.00` is sufficient.
 * provide the entry of a `title` of another row, such as `SSP2-Base`.
-  * If the run with this `title` is set to `start = 1`, then runs that point to this `title` are turned into “subsequent runs“ that will be started after the linked one is finished. Note that in this case, the REMIND code must not change until the `full.gms` file of the last subsequent run was generated. Don't `git checkout` other branches or manually edit REMIND files.
+  * If the run with this `title` is set to `start = 1`, then runs that point to this `title` are turned into “subsequent runs“ that will be started after the linked one is finished. Note that in this case, the REMIND code must not change until the `full.gms` file of the last subsequent run was generated.
   * If the run with this `title` is set to `start = 0` or does not exist in the `scenario_config_XYZ.csv` file, the function `configure_cfg` of [`start.R`](../start.R) searches in `./output` for folder with that title which contain a `fulldata.gdx` and whose `log.txt` states `REMIND run finished!`, and then picks the one with the latest date and time in the folder name. Appending a `_` to a `path_gdx…` entry, yielding for example `BAU_Nash_`, forces REMIND to take `fulldata.gdx` from earlier `BAU_Nash` runs, but avoids to be counted as a subsequent run. This way, you can quickly switch between the two options.
 
 The image above shows these possibilities used for `path_gdx_ref`. Run `RCP20` has a complete path specified but will not be executed because `start = 0`. `RCP37` provides only the folder name, `RCP26_subsequent` waits for `SSP2-Base` to be finished, and `RCP26_forceoldrun` selects the latest already finished `SSP2-Base` run in the output folder and starts immediately. If you set `start = 0` also for `SSP2-Base`, then `RCP26_subsequent` will also try to find an old run in a folder that looks like `SSP2-Base_YYYY-MM-DD_HH.MM.SS`.
```

Comment:
Could you add which behaviour is expected for `RCP37`? Start immediately if `SSP2-Base_2021-...` finished, otherwise turned into “subsequent runs“, right?

Reply:
no, as there is no scenario with that long `title`, it looks for a `fulldata.gdx` in this folder name and starts directly. If it doesn't find a file, `start.R` stops with an error message.